### PR TITLE
media-libs/libdvbpsi: version bump

### PR DESCRIPTION
**Pull request by seewpx**
Package-Manager: Portage-3.0.13, Repoman-3.0.2
Bug: https://bugs.gentoo.org/765136

### CODE REVIEW

File: media-libs/libdvbpsi/libdvbpsi-1.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (dm0-) [inline media-libs/libdvbpsi/libdvbpsi-1.3.3.ebuild:4]: Please update to EAPI 7, and change `DEPEND` to `BDEPEND` to install doc tools natively while cross-compiling.

Comment (seewpx) [reply media-libs/libdvbpsi/libdvbpsi-1.3.3.ebuild]: > Please update to EAPI 7, and change DEPEND to BDEPEND to install doc tools natively while cross-compiling.

Done! 
https://github.com/gentoo/gentoo/pull/19050/commits/f97e4c60f924abcfe8207ebe4e52632738ee259e

**Comment by thesamesam** [comment]
Thank you!

**Comment by thesamesam** [comment]
Merged in dc4096a9706df2a9a9e4d9e195761e23fc6787e6!

**Pull request closed without merging**